### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by Toshimasa-Matsumoto**
重要な点は、
・input
tabular、image、tabular+image
・task
分類(cls)、回帰(reg)、分類+期間(prog)

#### file
train.py -> modelsを見に行く
test.py -> modelsを見に行く
eval.py -> metricsを見に行く。clsならauc + roc fig, regならr2 + yy fig, progならc-indexを計算。
parameter.csv
work_all.sh

#### directory
・models/
 criterion.py
 dataloader.py
 env.py <- spilts以下のcsvの情報を使いやすいように整形している。
 loss.py 
 optimizer.py
 net.py <- netの部品を構成する。
 framework.py <-MLP, (CNN or ViT), MLP+(CNN or ViT)の大きな構造を決定する。
 options.py

・metrics/
 roc.py
 yy.py
 c-index.py

### 修正すること メモ まとめ
✔︎ Augmentationを次の４パターンから選ぶようにする。
　randaug, trivialaugwide, augmix, no
✔︎ materialsをNervus配下に置く。
✔︎ align_env.py の class NervusEnv をなくす。
✔︎ util.pyをなくす。(loggerは残す)
　もしその中で定義している関数が、必要ならば、その関数を使っているファイルに移す。
✔︎ train_option.py と test_option.py を1つにまとめる。
✔︎ single-label を multi-labelで label数=1 の場合と見なして扱う。
✔︎ dataloader, lossの計算 をマージする
✔︎ MLPは３層にする
✔︎ classificationで、accの計算なしにする
・testした結果のlikelihood.csvに、Filenameコラムを残す。
・uniq_idコラムは、likelihood.csvにいらない。 
・READMEの修正

### 作業したこと
train.py が動作するまでリファクタ済。

### CODE REVIEW

**Comment by Toshimasa-Matsumoto** [comment]
Moved parepamers/parameter.csv.
